packages/inferno/src/core/component.ts

```diff
@@ -143,6 +143,7 @@ export class Component<P = {}, S = {}> implements IComponent<P, S> {
   public $L: Function[] | null = null; // Current lifecycle of this component
   public $SVG: boolean = false; // Flag to keep track if component is inside SVG tree
   public $F: boolean = false; // Force update flag
+  public $MV: boolean = false; // Coordinating a move animation (set on parent node)
```

Comment:
this could be declared in inferno-animation component? I cant see any usages inside inferno for it

Reply:
Isn't it better to have it in core so one doesn't by mistake use it for something else?

Reply:
Yeah not a big deal